demo/run-osm-demo.sh

```diff
@@ -24,8 +24,8 @@ BOOKWAREHOUSE_NAMESPACE="${BOOKWAREHOUSE_NAMESPACE:-bookwarehouse}"
 CERT_MANAGER="${CERT_MANAGER:-tresor}"
 CTR_REGISTRY="${CTR_REGISTRY:-localhost:5000}"
 CTR_REGISTRY_CREDS_NAME="${CTR_REGISTRY_CREDS_NAME:-acr-creds}"
-CTR_TAG="${CTR_TAG:-latest}"
 DEPLOY_TRAFFIC_SPLIT="${DEPLOY_TRAFFIC_SPLIT:-true}"
+CTR_TAG=$(git rev-parse --short HEAD)
```

Comment:
Does the short SHA match the `github.sha` in the workflow? I thought github used the longer SHA.

Reply:
let me check. I also meant to make this the default not the actual value 🤦